query_plan.go

```diff
@@ -79,6 +82,58 @@ func (n *Node) Render() string {
 	return "\n" + tree.String()
 }
 
+type RenderedTreeWithStats struct {
+	Text         string
+	RowsTotal    string
+	Execution    string
+	LatencyTotal string
+}
+
+func (n *Node) RenderTreeWithStats() []RenderedTreeWithStats {
+	tree := treeprint.New()
+	renderTreeWithStats(tree, "", n)
+	var result []RenderedTreeWithStats
+	for _, line := range strings.Split(tree.String(), "\n") {
+		if line == "" {
+			continue
+		}
+
+		split := strings.Split(line, "\t")
+		if len(split) == 1 {
+			result = append(result, RenderedTreeWithStats{Text: line})
+			continue
+		}
+		branchText, protojsonText := split[0], split[1]
+
+		var value structpb.Value
+		err := protojson.Unmarshal([]byte(protojsonText), &value)
```

Comment:
We can narrow the scope for `err` with
```
if err := protojson.Unmarshal([]byte(protojsonText), &value); err != nil {
    ...
}
```

Reply:
Fixed in https://github.com/cloudspannerecosystem/spanner-cli/pull/66/commits/20d5d8a5f8338a46dd3c63b354f98ca2a448d0d0